tests/test-cases/Group23-VIC-Machine-Service/23-02-VCH-List.robot

```diff
@@ -111,3 +111,17 @@ Get VCH List Within Invalid Datacenter and Compute Resource
 
     Verify Return Code
     Verify Status Not Found
+
+    [Teardown]   Run Secret VIC Machine Delete Command   %{VCH-NAME}
+
+Empty VCH list returned
+    Get VCH List
+
+    Verify Return Code
+    Verify Status Ok
+
+    Get VCH ID    %{VCH-NAME}
+
+    Output Should Not Contain  .vchs[] | select(.name=="%{VCH-NAME}").admin_portal
```

Comment:
Also, since the output json contains empty VCH list, I guess it would be good to check if the length of `.vchs` field is indeed 0. 
(I'm not sure if we have `Output Length Should Be` keyword in `vic-machine-service-util.robot`, but it would probably be helpful to add one if we need it)